src/ngraph/runtime/gpu/gpu_memory_manager.cpp

```diff
@@ -80,6 +80,8 @@ void runtime::gpu::GPUMemoryManager::allocate()
             m_argspace_mem.back().ptr, m_buffered_mem.data(), m_buffer_offset);
         // add an empty node to the end of the list and zero offset
         m_argspace_mem.push_back({nullptr, 0});
+        m_buffered_mem.clear();
```

Comment:
Why reallocate & zero initialize?

Reply:
Here will release resource after use. for example, you might used 10GB buffer, you can release it and reallocate to initial size, which is much smaller, like 10MB.

Reply:
same as above for zero initialize.